services/api-serverless/api/requirements.txt

```diff
@@ -1 +0,0 @@
-numpy==1.23.3
```

Comment:
ok!

Reply:
I needed them to test the streamlit interface. Removing them would break the streamlit service, I suggest solving any conflicts when the separate PR is rebased and eventually merged.